### PR TITLE
enhancement: migrate to js

### PR DESCRIPTION
**Pull request by darrenvechain**

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -1,10 +1,9 @@
-import { HardhatUserConfig } from 'hardhat/config'
 import '@typechain/hardhat'
 import '@nomicfoundation/hardhat-ethers'
 import '@nomicfoundation/hardhat-chai-matchers'
 import '@nomicfoundation/hardhat-toolbox'
 
-const config: HardhatUserConfig = {
+const config = {
```
Comment (leszek-vechain) [inline hardhat.config.ts:6]: does it matter if it is js or ts file ?

File: src/test-env.js [hunk 2]
```diff
@@ -7,17 +7,10 @@ const nodeUrls = process.env.NODE_URLS
...
 
-export const E2eTestTags = [
-    'solo',
-    'default-private',
-    'testnet',
-    'mainnet',
-] as const
-
-export type E2eTestTag = (typeof E2eTestTags)[number]
+export const E2eTestTags = ['solo', 'default-private', 'testnet', 'mainnet']
```
Comment (leszek-vechain) [inline src/test-env.js:10]: just a q - did we also had all here ?

**Review by leszek-vechain — APPROVED** [review]
looks good

File: src/test-env.js (hunk 2 above)
Comment (darrenvechain) [reply src/test-env.js:10]: Nope, thats a param we can pass as an e2e tag

File: hardhat.config.ts (hunk 1 above)
Comment (darrenvechain) [reply hardhat.config.ts]: fixed

**Pull request merged**